src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -64,18 +69,26 @@ const AppointmentDetails = props => {
 
   const clinic = appointment && clinicName(appointment);
 
-  const preCheckInSubTitle = isPhoneAppointment ? (
-    <p data-testid="phone-appointment-subtitle" className="vads-u-margin--0">
-      {t('your-provider-will-call-you-at-your-appointment-time')}
-    </p>
-  ) : (
+  let preCheckInSubTitle = (
     <p
       data-testid="in-person-appointment-subtitle"
       className="vads-u-margin--0"
     >
       {t('please-bring-your-insurance-cards-with-you-to-your-appointment')}
     </p>
   );
+  if (is45MinuteReminderEnabled) {
+    <span data-testid="in-person-msg-confirmation">
```

Comment:
Is this part of this ticket? This would have to set preCheckInSubTitle like the isPhoneAppointment check below.

Reply:
don't forget to add the className that the others have.

Reply:
Updated that section and a test that goes along with it